internal/api/hooks.go

```diff
@@ -1,17 +1,36 @@
 package api
 
 import (
+	"bytes"
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
+	"io"
+	"net"
 	"net/http"
+	"net/http/httptrace"
+	"net/url"
+	"strings"
+	"time"
 
+	"github.com/gofrs/uuid"
+	standardwebhooks "github.com/standard-webhooks/standard-webhooks/libraries/go"
+	"github.com/supabase/auth/internal/conf"
+
+	"github.com/sirupsen/logrus"
 	"github.com/supabase/auth/internal/hooks"
 
 	"github.com/supabase/auth/internal/storage"
 )
 
-func (a *API) runHook(ctx context.Context, tx *storage.Connection, name string, input, output any) ([]byte, error) {
+const (
+	SymmetricSignaturePrefix = "v1,"
```

Comment:
Might be better to define a function to check if it's symmetric, or better yet one that creates a verifier object.

Reply:
moved the constant into a function in the `crypto` package